go/vt/vtgate/planbuilder/operator_transformers.go

```diff
@@ -689,24 +689,20 @@ func buildUpdateLogicalPlan(
 	return &primitiveWrapper{prim: e}, nil
 }
 
-func buildDeleteLogicalPlan(
-	ctx *plancontext.PlanningContext,
-	rb *operators.Route,
-	dmlOp operators.Operator,
-	hints *queryHints,
-) (logicalPlan, error) {
+func buildDeleteLogicalPlan(ctx *plancontext.PlanningContext, rb *operators.Route, dmlOp operators.Operator, stmt *sqlparser.Delete, hints *queryHints) (logicalPlan, error) {
```

Comment:
Why are we adding `stmt` to the function call, if we already have the ast in `del.AST`

Reply:
Never mind, I found the answer looking at the code further, you removed the AST from the delete operator

Reply:
nit: we can put the arguments on multiple lines as it was before